_query-dsl/full-text/query-string.md

```diff
@@ -39,9 +39,11 @@ You can use query string syntax in the following cases:
     ```
     {% include copy-curl.html %}
 
-1. In the Discover app of OpenSearch Dashboards, if you turn off DQL, as shown in the following image.
+1. In the Discover or Dashboard app of OpenSearch Dashboards, if you turn off DQL, as shown in the following image.
   ![Using query string syntax in OpenSearch Dashboards Discover]({{site.url}}{{site.baseurl}}/images/discover-lucene-syntax.png)
-  For more information, see [Discover]({{site.url}}{{site.baseurl}}/dashboards/discover/index-discover/).
+  
+  DQL and Query string query (Lucene) language are the two language options for the search bar in Discover and Dashboards. To compare these language options, see [Discover and Dashboard search bar]({{site.url}}{{site.baseurl}}/dashboards/index/#discover-and-dashboard-search-bar).
```

Comment:
I would switch the order of the languages so that you don't have the capitalization in the middle of the sentence.